app/Commands/EnableCommand.php

```diff
@@ -37,7 +40,8 @@ public function handle(): void
     public function enableableServices(): array
     {
         return collect((new Services)->all())->mapWithKeys(function ($fqcn, $shortName) {
-            return [$shortName => Str::afterLast($fqcn, '\\')];
+            $fullName = Str::of($fqcn)->replace(['MsS', 'Sql'], ['MS S', 'SQL']);
```

Comment:
Would you pull this out to a private method?

Thanks!

Reply:
Sure!